CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+
+**Step 2: Find an issue to work on**
+Please find an [issue](https://github.com/Terrastories/terrastories/issues) that you would like to take on and comment to assign yourself if no one else has done so already. All issues with the label `status: help wanted` are up for grabs! We will add the `status: claimed` label to the issue to mark it as assigned to you. Also, feel free to ask questions in the issues, and we will get back to you ASAP!
+
+**Step 3: Fork the repo**
+Click the "fork" button in the upper right of the Github repo page. A fork is a copy of the repository that allows you to freely explore & experiment without changing the original project. You can learn more about forking a repo in [this article](https://help.github.com/articles/fork-a-repo/).
+
+**Step 4: Create a branch**
+Checkout a new branch for your issue - this branch can be named anything, but we encourage the format  `XXX-brief-description-of-feature`  where  `XXX`  is the issue number.
+
+**Step 5: Happy Hacking!**
+Follow the instructions in the [Setup Document](https://github.com/Terrastories/terrastories/blob/master/documentation/DEVELOPMENT.md) to set up your local environment. Feel free to discuss any questions on the issues as needed, and we will get back to you! Don't forget to write some tests to verify your code. Commit your changes locally, using descriptive messages and please be sure to note the parts of the app that are affected by this commit.
+
+**Step 6: Pushing your branch and creating a pull request**
+Push your branch up and create a pull request! Please indicate which issue your PR addresses in the title.
+
+### Code Reviews & Pull Request Merging
+Once you've submitted a pull request, a core contributor will work with you on doing a code review (typically pretty minor unless it's a very significant PR). If the reviewer gives a ✅ to the PR merging, then huzzah! Merge into master! If your feature branch was in this main repository (and not forked), please delete your branch after it has been merged.
+
+### Stay Scoped
+Try to keep your PRs limited to one particular issue and don't make changes that are out of scope for that issue. If you notice something that needs attention but is out-of-scope, put a TODO, FIXME, or NOTE comment above it.
+
+### Work In Progress Pull Requests
+Sometimes we want to get a PR up there and going so that other people can review it or provide feedback, but maybe it's incomplete. This is OK, but if you do it, please tag your PR with an  `in-progress`  label so that we know not to review / merge it.
```

Comment:
Perhaps we can note that "Draft" PR's are also an option if they want to open one that isn't quite ready for review yet.